src/en/dynasty/src/eu/kanade/tachiyomi/extension/en/dynasty/DynastyDoujins.kt

```diff
@@ -64,7 +74,31 @@ class DynastyDoujins : DynastyScans() {
                 )
             }
         }
+        chapters.addAll(doujinChapterParse(document))
+
+        var hasNextPage = popularMangaNextPageSelector().let { selector ->
+            document.select(selector).first()
+        } != null
+
+        while (hasNextPage) {
+            page += 1
+            val doujinURL = document.location() + "?page=$page"
 
+            val newRequest = GET(doujinURL, headers)
+            val newResponse = client.newCall(newRequest).execute()
+
+            if (!newResponse.isSuccessful) {
+                retry += 1
+                if (retry > maxRetry) { break } else { continue }
```

Comment:
If you `continue`, you'll hit `page += 1` and actually request the next page. Are you sure about this?

Reply:
You are correct, this will lead to unintended behavior. The latest commit now completely aborts chapter parsing if the max retries are reached.